pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -82,6 +82,13 @@ stdenv.mkDerivation rec {
       stripLen = 1;
       extraPrefix = "share/extensions/";
     })
+
+    # Fix build with Poppler 21.11.0.
+    # https://gitlab.com/inkscape/inkscape/-/commit/5724c21b9cb7b6176a7b36ca24068b148c817e82
```

Comment:
```suggestion
```

Reply:
That was supposed to link to https://gitlab.com/inkscape/inkscape/-/merge_requests/3622